data/lib/core/storages.lua

```diff
@@ -1774,494 +1771,603 @@ Storage = {
+			DarkTrails = {},
+			GloothEngineerOutfits = {},
+			Oramond = {},
+			Rathleton = {},
+		},
+		U10_55 = { -- update 10.55 - Reserved Storages 74751 - 74800
+			Dawnport = {
+				VocationReward = 20000,
+				Questline = 20001,
+				GoMain = 20002,
+				TheLostAmulet = 20003,
+				TheStolenLogBook = 20004,
+				TheRareHerb = 20005,
+				TheDormKey = 20006,
+				StrangeAmulet = 20007,
+				TornLogBook = 20008,
+				HerbFlower = 20009,
+				MorriskTroll = 20010,
+				MorrisTrollCount = 20011,
+				MorrisGoblin = 20012,
+				MorrisGoblinCount = 20013,
+				MorrisMinos = 20014,
+				MorrisMinosCount = 20015
+			},
+			SanctuaryOfTheLizardGod = {
+				LizardGodTeleport = 20020,
+				LegionHelmet = 20021
+			},
+		},
+		U10_70 = { -- update 10.70 - Reserved Storages 74801 - 74950
+			HeroOfRathleton = {},
+			LionsRock = {},
+			TheCursedCrystal = {},
+			TheFireFeatheredSerpentWorldChange = {},
+			TwentyMilesBeneathTheSea = {},
+		},
+		U10_80 = { -- update 10.80 - Reserved Storages 74951 - 75200
+			AsuraPalace = {},
+			Cartography101 = {},
+			ChakoyaIcebergMiniWorldChange = {},
+			GrimvaleMineWorldChange = {},
+			Grimvale = {},
+			HiveOutpostMiniWorldChange = {},
+			JungleCampMiniWorldChange = {},
+			NightmareIslesMiniWorldChange = {},
+			NightmareTeddy = {},
+			PoacherCavesMiniWorldChange = {},
+			TheGreatDragonHunt = {},
+			TheLostBrother = {},
+			TheTaintedSouls = {},
+		},
+		U10_90 = { -- update 10.90 - Reserved Storages 75201 - 75350
+			FerumbrasAscension = {},
+			Krailos = {},
+			RiftWarriorOutfits = {},
+		},
+		U10_94 = { -- update 10.94 - Reserved Storages 75351 - 75450
+			HeartOfDestruction = {},
+		},
+		U11_02 = { -- update 11.02 - Reserved Storages 75451 - 75650
+			FestiveOutfits = {},
+			FirstDragon = {},
+			ForgottenKnowledge = {},
+		},
+		U11_40 = { -- update 11.40 - Reserved Storages 75651 - 75850
+			CultsOfTibia = {},
+			ThreatenedDreams = {},
+		},
+		U11_50 = { -- update 11.50 - Reserved Storages 75851 - 76000
+			DangerousDepths = {},
+			HiddenThreats = {},
+			MakeshiftWarriorOutfits = {},
+		},
+		U11_80 = { -- update 11.80 - Reserved Storages 76001 - 76100
+			BattleMageOutfits = {},
+			DiscovererOutfits = {},
+			MeasuringTibia = {},
+			TheSecretLibrary = {},
+		},
+		U12_00 = { -- update 12.00 - Reserved Storages 76101 - 76300
+			DreamWarriorOutfits = {},
+			TheDreamCourts = {
+				AndrewDoor = 14900
+			},
+		},
+		U12_02 = { -- update 12.02 - Reserved Storages 76301 - 76400
+			TibiaAnniversary = {},
+			WinterlightSolstice = {},
+		},
+		U12_15 = { -- update 12.15.8659 - Reserved Storages 76401 - 76450
+			GoldenOutfits = {},
+		},
+		U12_20 = { -- update 12.20 - Reserved Storages 76451 - 76650
+			GraveDanger = {},
+			HandOfTheInquisitionOutfits = {},
+			-- Kilmaresh = {}, done earlier in the file
+		},
+		U12_30 = { -- update 12.30 - Reserved Storages 76651 - 76850
+			FalconerOutfits = {},
+			FeasterOfSouls = {},
+			PoltergeistOutfits = {}
 		},
```

Comment:
I Will suggest here to keep updated with the last updates, to include this:

		U12_40 = { -- update 12.40 - Reserved Storages xxxx
			RevenantOutfits = {},
			SoulWar = {},
			TheOrderOfTheLion = {}
		},
		U12_60 = { -- update 12.60 - Reserved Storages xxxx
			APiratesTail = {},
			RascoohanOutfits = {}
		},
		U12_80 = { -- update 12.80 - Reserved Storages xxxx
			RoyalCostumeOutfits = {},
		},